src/sentry/api/endpoints/sentry_app_publish_request.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry import options
+from sentry import features
+from sentry.api.bases.sentryapps import SentryAppBaseEndpoint
+from sentry.utils.email import send_mail
+from sentry.utils import json
+
+
+class SentryAppPublishRequestEndpoint(SentryAppBaseEndpoint):
+    def put(self, request, sentry_app):
```

Comment:
This should probably be a `POST` since we're _creating_ a publish request, not updating one.